tests/Monolog/RecordFactory.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Tests\Monolog;
+
+use DateTime;
+use DateTimeImmutable;
+use Monolog\Logger;
+use Monolog\LogRecord;
+
+class RecordFactory
+{
+    public static function create($message, $level, string $channel, array $context, array $extra, ?DateTime $datetime = null)
```

Comment:
Why `$message` has no typehint? Also, it looks like we never use the `$datetime` argument when calling this function, so we can remove it

Reply:
> Why $message has no typehint?

I have just added the missing type hints.